source/jobs/sync-jobs.txt

```diff
@@ -40,6 +40,19 @@ Recoverability
 
 .. include:: /includes/fact-cdc-recoverable.rst
 
+Behavior
+--------
+
+Sync jobs are non-impodent by default. Relational Migrator inserts new 
```

Comment:
Spelling mistake "impodent"

Reply:
Thanks, looks like I had blinders on after swapping out the term accidentally.